cmd/brew-test-bot.rb

```diff
@@ -541,6 +543,8 @@ def download
         puts "#{@tap} HEAD #{tap_revision}"
```

Comment:
This (and line 538) are already outputting the revision of the tap. I'd suggest these messages be combined or adjusted instead of duplicated.